Time to make the long provisioning warning be a bear. (and also provide more accurate information) (#2506)
Varying-Vagrant-Vagrants/VVV: CHANGELOG.md

## 3.7.2 ( 2021 )

> * Updated message on full provision to be more clear
> * Added mascot to full provision message

Reviewer: I would use bear instead of mascot, as it's used in other places across the changelog, and also join it in a single line.
```suggestion
* Added a new bear to full provision message, and updated message to be more clear
```

Author: 👍 Sounds good to me. Asked about the name because I was thinking about using that, but matching what's been used before is more important

🎉

Reviewer 2: VVV bear? The bear was more me adding a bit of whimsy so that it was obvious when a vagrant up or provision had succeeded, users weren't sure if it was still provisioning or what to do next. A big ascii teddy bear seemed like a great way to indicate it had finished, and successfully

Author: Well now we need to give them a name! I love that lil bear

Reviewer 2: Or her! I've asked the bear and have yet to get an answer back. Perhaps one day in the dashboard you can customize your own VVV bear but that's a long way off
